src/app/WriteClient.h

```diff
@@ -41,6 +41,8 @@
 #include <system/SystemPacketBuffer.h>
 #include <system/TLVPacketBufferBackingStore.h>
 
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
```

Comment:
this should be adjusted after #38151 ... this pulls in a huge dependency just for OTA